### ISSUE TITLE
edges2shoes: the output seemed the almost the same as the input image

### ISSUE THREAD

**Issue by tangmin0714** [phillipi/pix2pix#168]
I tried the edges2shoes, setting batch size to 4 and other parameters as default. After a few epochs, the generator loss did not decrease any more, and after training more than 80 epochs, the output were just look very similar to the input image. Is it normal, and could you give me some advice? 
Thanks a lot for your kindness!

**Comment by junyanz** [comment]
This is not normal. Could you share with us more details? Your command line for example. 
Have you tried the PyTorch [code](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix)?

**Comment by tangmin0714** [comment]
<img width="843" alt="屏幕快照 2019-07-10 下午2 41 22" src="https://user-images.githubusercontent.com/12481714/60946816-d0922e00-a321-11e9-9bad-f4315cb560d2.png">
wow! It's a surprise you answered my question. I tried another model it seems perform better

**Comment by tangmin0714** [comment]
the effect of the GAN model trained by us seems like the picture above

**Comment by tangmin0714** [comment]
I am a bit confused if we can try some transformation like edges to shoes/handbags with the SPADE model proposed in your paper "Semantic Image Synthesis with Spatially-Adaptive Normalization"

**Comment by junyanz** [comment]
I guess so. Feel free to try it.
